Content.Shared/Blocking/BlockingSystem.cs

```diff
@@ -187,6 +229,15 @@ private void BlockingShutdownHelper(EntityUid uid, BlockingComponent component,
         if (component.IsBlocking)
             StopBlocking(uid, component, user);
 
+        foreach (var shield in _handsSystem.EnumerateHeld(user))
+        {
+            if (HasComp<BlockingComponent>(shield) && TryComp<BlockingUserComponent>(user, out var blockingUserComponent))
+            {
+                blockingUserComponent.BlockingItem = shield;
+                return;
+            }
+        }
+
```

Comment:
i feel like `StopBlocking` should be checking this. won't that just get called earlier and then this code will run and not actually fix anything

Reply:
I'll also post what I said in discord here: It shouldn't be in stop blocking because this should only happen if they break/drop/unequip the other shield. If it's in stop blocking then anytime someone lowers their shield, then it swaps for some reason if they have another shield